ejenti/pulse_modules/tasksTrigger.py

```diff
@@ -252,10 +248,7 @@ def clean_md5_by_job_name(job_name):
 
         # prepare MD5 folder
         if os.path.exists(md5_folder):
-            if os.path.isdir(md5_folder):
-                shutil.rmtree(md5_folder)
-                os.makedirs(md5_folder)
-            elif os.path.isfile(md5_folder):
+            if os.path.isfile(md5_folder):
                 os.remove(md5_folder)
         else:
             os.makedirs(md5_folder)
```

Comment:
Depends on method functionality, it could `return` after `makedirs` because it's an empty folder and doesn't need to clean anything.